workflows/theiacov/wf_theiacov_illumina_pe.wdl

```diff
@@ -227,8 +230,8 @@ workflow theiacov_illumina_pe {
             num_reads_raw2 = read_QC_trim.fastq_scan_raw2,
             num_reads_clean1 = read_QC_trim.fastq_scan_clean1,
             num_reads_clean2 = read_QC_trim.fastq_scan_clean2,
-            kraken_human = read_QC_trim.kraken_human,
-            kraken_human_dehosted = read_QC_trim.kraken_human_dehosted,
+            kraken_human = read_QC_trim.kraken2_human,
```

Comment:
could you update the qc_check_task to use `kraken2` as well for consistency? probably going to have to make this change in the export_taxon_table task as well

Reply:
kraken2 is kraken2 _everywhere_ now